### ISSUE TITLE
Error in OrbitIntegralAction

### ISSUE THREAD

**Issue by fingolfin** [gap-packages/polycyclic#21]
The following issue was originally reported at https://github.com/gap-system/gap/issues/2179 and I suspect this is related to issue #15 and possibly also #16. Consider this code:
```gap
G:=ExamplesOfSomePcpGroups(8);
mats:=Representation(Collector(G));;
e:=[8,-4,5,2,13,-17,9];;
f := e * MappedVector( [ -2, 2, 0, 5, 5 ], mats );
o := OrbitIntegralAction( G, mats, e, f );  # should return true
```
This leads to the following unexpected error:
```
gap> G:=ExamplesOfSomePcpGroups(8);
Pcp-group with orders [ 0, 0, 0, 0, 0 ]
gap> mats:=Representation(Collector(G));;
gap> e:=[8,-4,5,2,13,-17,9];;
gap> f := e * MappedVector( [ -2, 2, 0, 5, 5 ], mats );
[ 8, -4, 19, 34, 51, -17, 5 ]
gap> o := OrbitIntegralAction( G, mats, e, f );  # should return true
Error, no method found! For debugging hints type ?Recovery from NoMethodFound
Error, no 1st choice method found for `[]:=' on 3 arguments
The 2nd argument is 'fail' which might point to an earlier problem
 at /Users/mhorn/Projekte/GAP/gap.github/lib/methsel2.g:250 called from
v[l] at /Users/mhorn/Projekte/GAP/repos/pkg/pkg/pkg/polycyclic.remote-hg/gap/matrix/rowbases.gi:85 called from
CoefficientsByNHSEB( vec, hom ) at /Users/mhorn/Projekte/GAP/repos/pkg/pkg/pkg/polycyclic.remote-hg/gap/matrix/rowbases.gi:106 called from
ImageByNHSEB( x, nath ) at /Users/mhorn/Projekte/GAP/repos/pkg/pkg/pkg/polycyclic.remote-hg/gap/action/orbstab.gi:376 called from
func( C[i] ) at /Users/mhorn/Projekte/GAP/gap.github/lib/coll.gi:653 called from
List( derS, function ( x )
      return ImageByNHSEB( x, nath );
  end ) at /Users/mhorn/Projekte/GAP/repos/pkg/pkg/pkg/polycyclic.remote-hg/gap/action/orbstab.gi:376 called from
...  at *stdin*:9
you can 'quit;' to quit to outer loop, or
you can 'return;' to continue
brk>
```

**Comment by fingolfin** [comment]
Some more debug output:
```
gap> SetInfoLevel(InfoIntStab,5);
gap> o := OrbitIntegralAction( G, mats, e, f );  # should return true
#I  reducing by orbit-stabilizer mod 3
#I  determining 3-congruence subgroup
#I  computing module series
#I  constructing block orbit-stabilizer
#I
#I    consider layer 1 of dim 2
#I    computing kernel of linear action
#I    spaces: [ 1, 1 ]
#I    computing orbit by irreducible action
#I    spaces: [ 1 ]
#I    computing orbit by irreducible action
#I
#I    consider layer 2 of dim 3
#I    computing kernel of linear action
#I    spaces: [ 1, 1, 1 ]
#I    computing orbit by irreducible action
#I    translation orbit: [ [ 12 ] ]
#I
#I    consider layer 1 of dim 2
#I    computing kernel of linear action
#I    spaces: [ 1, 1 ]
#I    computing orbit by irreducible action
#I    spaces: [ 1 ]
#I    computing orbit by irreducible action
#I
#I    consider layer 2 of dim 3
#I    computing kernel of linear action
#I    spaces: [ 1, 1, 1 ]
#I    computing orbit by irreducible action
#I    translation orbit: [ [ 12 ] ]
#I    spaces: [ 1, 1 ]
#I    computing orbit by irreducible action
#I    spaces: [ 1 ]
#I    computing orbit by irreducible action
#I    translation orbit: [ [ 12 ] ]
#I
#I    consider layer 3 of dim 2
Error, no method found! For debugging hints type ?Recovery from NoMethodFound
Error, no 1st choice method found for `[]:=' on 3 arguments
The 2nd argument is 'fail' which might point to an earlier problem
 at /Users/mhorn/Projekte/GAP/gap.github/lib/methsel2.g:250 called from
v[l] at GAPROOT/pkg/polycyclic/gap/matrix/rowbases.gi:85 called from
CoefficientsByNHSEB( vec, hom ) at GAPROOT/pkg/polycyclic/gap/matrix/rowbases.gi:106 called from
ImageByNHSEB( x, nath ) at GAPROOT/pkg/polycyclic/gap/action/orbstab.gi:376 called from
func( C[i] ) at /Users/mhorn/Projekte/GAP/gap.github/lib/coll.gi:653 called from
List( derS, function ( x )
      return ImageByNHSEB( x, nath );
  end ) at GAPROOT/pkg/polycyclic/gap/action/orbstab.gi:376 called from
OrbitCongruenceAction( K, actK, a, b, ser ) at GAPROOT/pkg/polycyclic/gap/action/orbstab.gi:665 called from
orbfun( K, actK, pt, orbit[j] ) at GAPROOT/pkg/polycyclic/gap/action/orbstab.gi:447 called from
FindPosition( orbit, op( n[1], mats[i] ), K, actK, orbfun ) at GAPROOT/pkg/polycyclic/gap/action/orbstab.gi:487 called from
ExtendOrbitStabilizer( e, K, actK, S, actS, orbf, OnRight ) at GAPROOT/pkg/polycyclic/gap/action/orbstab.gi:672 called from
<function "OrbitIntegralAction">( <arguments> )
 called from read-eval loop at *errin*:1
```

**Comment by fingolfin** [comment]
Fixed by commit 0c48cc92d2a9e002b682a4e365107c50977a1f70
